### PR TITLE
Refactor(kclvm-win): rm python39 for kclvm-windows.

### LINKED ISSUE #379: [Enhancement] KCLVM windows artifact optimization.
## Enhancement

We need to further optimize KCLVM windows artifacts as follows:

+ [x] Size optimization. -> ~50M @zong-zhe #382 
+ [x] Scoop support for windows binaries @Peefy https://github.com/KusionStack/kcl-lang.io/pull/32
+ [x] The stability of kcl windows version @zong-zhe @NeverRaR @Peefy #403
+ [x] kclvm windows artifact embedded in the kclvm-artifact-go repo @NeverRaR https://github.com/KusionStack/kclvm-artifact-go/releases/tag/v0.4.5
+ [x] Windows directly from the web @Peefy https://github.com/KusionStack/KCLVM/releases/tag/v0.4.5

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

fix #379.

#### 2. What is the scope of this PR (e.g. component or file name):

kclvm-windows

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

remove the python packaged in kclvm-windows.

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: scripts/build-windows/kcl-doc.go [hunk 1]
```diff
@@ -6,54 +6,85 @@
...
-)
-
-// int Py_Main(int argc, wchar_t **argv)
 func Py_Main(args []string) int {
-	c_args := make([]*uint16, len(args)+1)
-	for i, s := range args {
-		c_args[i] = syscall.StringToUTF16Ptr(s)
+	inputPath, err := os.Executable()
+	if err != nil {
+		fmt.Println("Input path does not exist")
```
Comment (Peefy) [inline scripts/build-windows/kcl-doc.go:28]: Should the promotion information be output to stderr?

File: scripts/build-windows/kcl-fmt.go [hunk 2]
```diff
@@ -6,54 +6,85 @@
...
 }
 
-func findKclvm_dllPath() string {
-	kclvmName := "python39.dll"
+func Install_Kclvm(installed_path string) {
+	// Check if Python3 is installed
+	cmd := exec.Command("cmd", "/C", "where python3")
+	_, err := cmd.Output()
+	if err != nil {
+		fmt.Println("Python3 is not installed, details: ", err)
```
Comment (Peefy) [inline scripts/build-windows/kcl-fmt.go:55]: Should the promotion information be output to stderr? All other places where `os.Exit(1)` appears should be considered this way.

File: scripts/build-windows/kcl-doc.go [hunk 3]
```diff
@@ -6,54 +6,85 @@
...
 }
 
-func findKclvm_dllPath() string {
-	kclvmName := "python39.dll"
+func Install_Kclvm(installed_path string) {
+	// Check if Python3 is installed
+	cmd := exec.Command("cmd", "/C", "where python3")
+	_, err := cmd.Output()
+	if err != nil {
+		fmt.Println("Python3 is not installed, details: ", err)
```
[the remark this answers is not in the document]
Comment (zong-zhe) [reply scripts/build-windows/kcl-doc.go]: fixed

File: scripts/build-windows/kcl-fmt.go (hunk 2 above)
Comment (zong-zhe) [reply scripts/build-windows/kcl-fmt.go]: fixed

File: scripts/build-windows/kcl-doc.go (hunk 1 above)
Comment (zong-zhe) [reply scripts/build-windows/kcl-doc.go]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Review by NeverRaR — APPROVED** [review]
lgtm

**Pull request merged**